{"hierarchy":{"paths":[["doc:\/\/com.estelapasjak.Mdots\/documentation\/MDots","doc:\/\/com.estelapasjak.Mdots\/documentation\/MDots\/AuthError"]]},"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.estelapasjak.Mdots\/documentation\/MDots\/AuthError\/errorDescription"},"primaryContentSections":[{"declarations":[{"languages":["swift"],"platforms":["iOS"],"tokens":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"errorDescription"},{"text":": ","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"kind":"text","text":"? { "},{"text":"get","kind":"keyword"},{"kind":"text","text":" }"}]}],"kind":"declarations"},{"kind":"content","content":[{"type":"heading","level":2,"text":"Discussion","anchor":"discussion"},{"type":"paragraph","inlineContent":[{"text":"This property provides a localized description for each error case, making it easier to","type":"text"},{"text":" ","type":"text"},{"text":"understand what went wrong during the authentication process.","type":"text"}]}]}],"variants":[{"paths":["\/documentation\/mdots\/autherror\/errordescription"],"traits":[{"interfaceLanguage":"swift"}]}],"metadata":{"modules":[{"name":"MDots"}],"roleHeading":"Instance Property","fragments":[{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"text":"errorDescription","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"},{"kind":"text","text":"?"}],"symbolKind":"property","title":"errorDescription","externalID":"s:5MDots9AuthErrorO16errorDescriptionSSSgvp","role":"symbol"},"sections":[],"abstract":[{"text":"A description of the error, suitable for displaying to the user.","type":"text"}],"kind":"symbol","schemaVersion":{"patch":0,"minor":3,"major":0},"references":{"doc://com.estelapasjak.Mdots/documentation/MDots":{"kind":"symbol","abstract":[],"identifier":"doc:\/\/com.estelapasjak.Mdots\/documentation\/MDots","url":"\/documentation\/mdots","type":"topic","images":[{"identifier":"new-icon.png","type":"icon"}],"title":"MDots","role":"collection"},"doc://com.estelapasjak.Mdots/documentation/MDots/AuthError":{"navigatorTitle":[{"text":"AuthError","kind":"identifier"}],"title":"AuthError","url":"\/documentation\/mdots\/autherror","kind":"symbol","fragments":[{"text":"enum","kind":"keyword"},{"kind":"text","text":" "},{"text":"AuthError","kind":"identifier"}],"identifier":"doc:\/\/com.estelapasjak.Mdots\/documentation\/MDots\/AuthError","role":"symbol","type":"topic","abstract":[{"type":"text","text":"An enumeration representing various authentication errors that can occur."}]},"doc://com.estelapasjak.Mdots/documentation/MDots/AuthError/errorDescription":{"abstract":[{"text":"A description of the error, suitable for displaying to the user.","type":"text"}],"fragments":[{"text":"var","kind":"keyword"},{"kind":"text","text":" "},{"text":"errorDescription","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"text":"?","kind":"text"}],"kind":"symbol","title":"errorDescription","url":"\/documentation\/mdots\/autherror\/errordescription","type":"topic","role":"symbol","identifier":"doc:\/\/com.estelapasjak.Mdots\/documentation\/MDots\/AuthError\/errorDescription"},"new-icon.png":{"variants":[{"traits":["1x","light"],"url":"\/images\/new-icon.png"}],"type":"image","identifier":"new-icon.png","alt":"The logo representing the MDot sensor application."}}}